ipa-core/src/ff/boolean_array.rs

```diff
@@ -917,40 +920,34 @@ macro_rules! boolean_array_impl {
 }
 
 macro_rules! boolean_array_impl_large {
-    ($modname:ident, $name:ident, $bits:tt, $deser_type:tt, $arraylength:ty) => {
+    ($modname:ident, $name:ident, $bits:tt, $deser_type:tt, $bytelength:ty, $wordlength:ty) => {
         boolean_array_impl!($modname, $name, $bits, $deser_type);
 
         $crate::const_assert!(
-            $bits <= 256,
-            "Large BooleanArrays only support up to 256 bits."
+            $bits % 8 == 0,
+            "Large BooleanArrays only support whole bytes."
         );
 
-        // used to convert into Fp25519
-        impl From<(u128, u128)> for $name {
```

Comment:
Use of this for `Fp25519` was removed some time ago. The only remaining use was a few tests in `dzkp_validator`, which could easily be changed to use something else.